graphql_compiler/query_formatting/common.py

```diff
@@ -63,6 +57,8 @@ def validate_argument_type(expected_type, value):
             raise GraphQLInvalidArgumentError(u'Attempting to represent a non-bool as a bool: '
                                               u'{} {}'.format(type(value), value))
     elif GraphQLDecimal.is_same_type(expected_type):
+        # Special case: in Python, isinstance(True, int) returns True.
+        # Safeguard against this with an explicit check against bool type.
```

Comment:
Maybe preface this comment with the types we'd like to support: `int, float, Decimal`, but not `bool`. Then this explanation makes sense.

Reply:
This is my only concern, otherwise this is good to go.